pkg/arvo/sys/hoon.hoon

```diff
@@ -10996,6 +10997,11 @@
           [%unit *]
         =^  cox  gid  $(q.ham q.q.ham)
         :_(gid [%rose [" " (weld (trip p.q.ham) "(") ")"] cox ~])
+      ::
+          [%name *]
+        :_  gid
+        ?@  p.q.ham  (cat 3 '#' mark.p.q.ham)
+        (rap 3 '#' auth.p.q.ham (spat type.p.q.ham) ~)
```

Comment:
Might want to have a separator between the name and path in the `^` case" something like `#name+/some/path` or `#name:/some/path` instead of `#name/some/path`.